google/cloud/storage/benchmarks/storage_latency_analysis.R

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env Rscript
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require(ggplot2)
```

Comment:
This is all basically Greek to me, but I trust that this makes sense :P

Reply:
Yeah... I tried to use Python: pandas, seaborn, scipy, oh my... I am sure they are great, but I could not figure it out, while R I know.  Created #1135 to remind myself we should clean this up.